GiniVision/Classes/Core/CapturedImagesStackView.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  CapturedImagesStackView.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 4/25/18.
+//
+
+import UIKit
+
+final class CapturedImagesStackView: UIView {
+    
+    enum State {
+        case filled(count: Int, lastImage: UIImage), empty
+    }
+    
+    let thumbnailSize: CGSize = {
+        if UIDevice.current.isIpad {
+            return CGSize(width: 40, height: 60)
```

Comment:
Can't you get away with not hardcoding this? Just infer it from the view size.

Reply:
No, because I want them to be that size.

Reply:
Maybe I could make it variable depending on container height (for iPhone) or width (for iPad), but due to time constraints and the complexity that this would add to the class, I will keep it as it is now.